tests/test-config.js

```diff
@@ -1,5 +1,73 @@
 /*!
- * Copyright 2023 Digital Bazaar, Inc. All Rights Reserved
+ * Copyright 2023-2024 Digital Bazaar, Inc. All Rights Reserved
```

Comment:
We can also drop the "All Rights Reserved" text, fwiw. It's ancient...and means things are under copyright. Something I missed removing earlier.

Reply:
appended this as an additional task to this issue: https://github.com/w3c/vc-di-ecdsa-test-suite/issues/27